eng/pipelines/publish.yml

```diff
@@ -69,6 +73,18 @@ jobs:
                 /p:NuGetApiKey=$(dotnet-myget-org-api-key)
         displayName: Push to myget.org
 
+      - script: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 build.proj
+                -warnaserror:0 -ci
+                /t:UpdatePublishedVersions
+                /p:GitHubUser=dotnet-helix-bot
+                /p:GitHubEmail=dotnet-helix-bot@microsoft.com
+                /p:GitHubAuthToken=$(AccessToken-dotnet-build-bot-public-repo)
+                /p:VersionsRepoOwner=dagood
```

Comment:
Should be `dotnet`. I think the cause is the same as the issue with user/email--for this, I had my fork as the definition default for testing purposes.

Reply:
I just verified this one, and it is indeed overridden to dotnet. Will update this one.